api/v1beta1/grpc_disruption.go

```diff
@@ -45,20 +45,20 @@ var ErrorMap = map[string]codes.Code{
 type GRPCDisruptionSpec struct {
 	// +kubebuilder:validation:Minimum=1
 	// +kubebuilder:validation:Maximum=65535
-	Port      int                  `json:"port"`
-	Endpoints []EndpointAlteration `json:"endpoints"`
+	Port      int                  `json:"port" validate:"gte=1,lte=65535"`
+	Endpoints []EndpointAlteration `json:"endpoints" validate:"dive"`
```

Comment:
What dive means? Recursively?

Reply:
Yes! Nested structs will be recursed automatically, but slices and maps need to be tagged with "dive" explicitly for validation to occur on each item. Where's the best place to explain things like this? In CONTRIBUTING.md?